models/models.js

```diff
@@ -29,12 +29,6 @@ exports.fetchAllArticles = () =>{
 
 exports.fetchArticleIdComments = (id) =>{
     return db.query(`Select * From comments WHERE comments.article_id = $1 ORDER BY created_at DESC`,[id]).then(({rows}) =>{
```

Comment:
Best practise to make SQL keywords clear by capitalising them, i.e. SELECT * FROM